examples/validators/gatekeeper-validate/invalid-configmap/README.md

```diff
@@ -0,0 +1,73 @@
+# gatekeeper-validate: invalid configmap
+
+## Overview
+
+This example demonstrates how to validate config maps using a constraint.
+
+There are 3 resources: a ConstraintTemplate, a K8sBannedConfigMapKeysV1 and a
+ConfigMap.
+The constraint disallows `private_key` to be used as a key in the config map.
```

Comment:
nit: should read like "The constraint disallows using `private_key` as a key in the config map."